src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/nonemptyatclausedescription/Example2.java

```diff
@@ -0,0 +1,31 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="NonEmptyAtclauseDescription">
+      <property name="javadocTokens" value="PARAM_LITERAL,THROWS_LITERAL"/>
+    </module>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.javadoc.nonemptyatclausedescription;
+
+// xdoc section -- start
+class Example2 {
+
+  /**
+   * Some summary.
+   *
+   * @param a Some description
+   * @param b
+   * @deprecated
+   * @throws Exception
+   * @return
+   */
+  public void testMethod(){
+    // violation 6 lines above 'At-clause should have a non-empty description'
+    // violation 5 lines above 'At-clause should have a non-empty description'
+    // @deprecated ignored as not mentioned in javadocTokens
```

Comment:
Please put this one line up.
To keep `violatiom 6 lines above` for all lines of violation, to be same as in previous example

Reply:
Done